### PR TITLE
pr commercial

### PR DESCRIPTION
**Pull request by qianmoQ**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation (changelog entry is not required)

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
...

Detailed description / Documentation draft:
...

> By adding documentation, you'll allow users to try your new feature immediately, not when someone else will have time to document it later. Documentation is necessary for all features that affect user experience in any way. You can add brief documentation draft above, or add documentation right into your patch as Markdown files in [docs](https://github.com/ClickHouse/ClickHouse/tree/master/docs) folder.

> If you are doing this for the first time, it's recommended to read the lightweight [Contributing to ClickHouse Documentation](https://github.com/ClickHouse/ClickHouse/tree/master/docs/README.md) guide first.

> Information about CI checks: https://clickhouse.tech/docs/en/development/continuous-integration/

### CODE REVIEW

File: docs/zh/commercial/cloud.md [hunk 1]
```diff
@@ -3,51 +3,60 @@ toc_priority: 1
 toc_title: 云
 ---
 
-# ClickHouse 云服务提供商 {#clickhouse-cloud-service-providers}
+# ClickHouse云服务提供商 {#clickhouse-cloud-service-providers}
```
Comment (sundy-li) [inline docs/zh/commercial/cloud.md:6]: I think it's better to add space between Chinese and English.

Refer: https://github.com/vinta/pangu.js

File: docs/zh/commercial/cloud.md [hunk 2]
```diff
@@ -6,48 +6,57 @@ toc_title: 云
 # ClickHouse 云服务提供商 {#clickhouse-cloud-service-providers}
 
 !!! info "注意"
-    如果您已经推出具有托管 ClickHouse 服务的公共云，请随时[提交一个 pull request](https://github.com/ClickHouse/ClickHouse/edit/master/docs/en/commercial/cloud.md) 将其添加到以下列表。
+    如果您已经推出具有托管ClickHouse服务的公共云，请随时[提交一个 pull request](https://github.com/ClickHouse/ClickHouse/edit/master/docs/en/commercial/cloud.md)将其添加到以下列表。
 
-## Yandex 云 {#yandex-cloud}
+## Yandex云 {#yandex-cloud}
```
Comment (sundy-li) [inline docs/zh/commercial/cloud.md:11]: Seems there are many other places that remove the space.

**Comment by alexey-milovidov** [comment]
As I understand, the main change is to move DBM from commercial to free.
Ok, though I did not check that it's actually free...

**Comment by qianmoQ** [comment]
> As I understand, the main change is to move DBM from commercial to free.
> Ok, though I did not check that it's actually free...

This is free. I filled in the wrong place

**Pull request merged**